src/cli.rs

```diff
@@ -358,3 +362,25 @@ impl FromStr for RequestItem {
         }
     }
 }
+
+#[derive(Debug, PartialEq, Clone)]
+pub enum VerifyHttps {
+    Yes,
+    No,
+    PrivateCerts(String),
+}
+
+impl FromStr for VerifyHttps {
+    type Err = Error;
+    fn from_str(verify: &str) -> Result<VerifyHttps> {
+        match verify.to_lowercase().as_str() {
+            "no" => Ok(VerifyHttps::No),
+            "yes" => Ok(VerifyHttps::Yes),
```

Comment:
`true` and `false` are also valid values in httpie. I don't know if @ducaale is aiming for an interface that is as close as possible to httpie's, but if he is, it might make sense to replicate that behavior here.

Reply:
I've added `"true"` and `"false"`as valid value.

I can remove those if it is not @ducaale aiming for.

Reply:
I believe most of the current `ht` users are coming from `httpie`, so it's nice to have a compatible interface to reduce friction. In the future, I am planning to list any divergence between `ht` and `httpie` in the `README.md` file.